static/src/javascripts/bootstraps/media.js

```diff
@@ -236,6 +240,41 @@ define([
         this.one(constructEventName('preroll:play', player), events.init.bind(player));
     }
 
+    function adSkipCountdown(skipTimeout) {
+        var player = this,
+            events =  {
+                update: function () {
+                    var skipTime = parseInt((skipTimeout - this.currentTime()).toFixed(), 10);
+                    if (skipTime > 0) {
+                        $('.js-skip-remaining-time', this.el()).text(skipTime);
+                    } else if (!skipTime) {
+                        $('.vjs-ads-overlay-skip-countdown', this.el())
+                            .html('<button class="vjs-ads-overlay-skip-button" data-link-name="vjs-skip-ads-button"><i class="i i-play-icon-grey skip-icon"></i>' +
```

Comment:
The `data-link-name` can be a more human friendly string as it's just used in omniture, such as: `Skip advert`